Lusas_Adapter/LusasAdapter.cs

```diff
@@ -56,15 +56,15 @@ public partial class LusasV17Adapter : BHoMAdapter
         /**** Constructors                              ****/
         /***************************************************/
 #if Debug18 || Release18
-        public LusasV18Adapter(string filePath, LusasConfig lusasConfig = null, bool active = false)
+        public LusasV18Adapter(string filePath, LusasSettings lusasSettings = null, bool active = false)
 #elif Debug19 || Release19
-        public LusasV19Adapter(string filePath, LusasConfig lusasConfig = null, bool active = false)
+        public LusasV19Adapter(string filePath, LusasSettings lusasSettings = null, bool active = false)
 #elif Debug191 || Release191
-        public LusasV191Adapter(string filePath, LusasConfig lusasConfig = null, bool active = false)
+        public LusasV191Adapter(string filePath, LusasSettings lusasSettings = null, bool active = false)
 #elif Debug200 || Release200
-        public LusasV200Adapter(string filePath, LusasConfig lusasConfig = null, bool active = false)
+        public LusasV200Adapter(string filePath, LusasSettings lusasSettings = null, bool active = false)
 #else
-        public LusasV17Adapter(string filePath, LusasConfig lusasConfig = null, bool active = false)
+        public LusasV17Adapter(string filePath, LusasSettings lusasSettings = null, bool active = false)
```

Comment:
Each of these need a previous version attribute, see:
https://bhom.xyz/documentation/Development/Versioning/versioning-guide/#changes-to-methods

Reply:
Added. Still not able to get it working properly though.